example/config/.mocharc.yml

```diff
@@ -6,8 +6,8 @@ check-leaks: false
 color: true
 delay: false
 diff: true
-exclude:
-  - /path/to/some/excluded/file
+ignore:
```

Comment:
If renamed, it should move to L28.

(and move `v8-stack-trace-limit` to L45)

Reply:
+1